cfn_template/ec2.go

```diff
@@ -34,35 +34,36 @@ func ELBToInstance(vpc, https bool, elbName, elbSecurityGroup string) map[string
 
 	if vpc {
 
-		sgIngress = []interface{}{
-			map[string]interface{}{
+		if !httpsOnly {
```

Comment:
A thought: the `httpsPort` and `!httpsOnly` blocks seem to be duped aside from the security group id, maybe break that out into a function and pass the security group info in?